meta-xt-domx/recipes-extended/xen/files/xt-defaults.cfg

```diff
@@ -0,0 +1,6 @@
+CONFIG_XSM=y
+CONFIG_TEE=y
+CONFIG_OPTEE=y
+CONFIG_IPMMU_VMSA=y
```

Comment:
Do we really need this in common part? This is R-Car specific.

Reply:
I don't know
```
do_configure:append() {
    oe_runmake xt_defconfig
```
Does the same but non-yocto way

Reply:
You are adding a new file to the common part of build system. Prior to your change there was no config file at all. 

Probably Xen repository used by Gen3/Gen4 products include those changes, but previously they were defined in products. But this does not answer my question

Reply:
What this do_configure:append() does in common part? Probably this recipe should be moved to rcar layers?

Reply:
Well, yes. This particular call to `xt_defconfig` should go into meta-xt-rcar

Reply:
Any way we moving that to xt-rcar doesn't fix main problem of this - double configuration in do_configure step